weave

```diff
@@ -1023,12 +1023,13 @@ case "$COMMAND" in
         collect_cidr_args "$@"
         shift $CIDR_COUNT
         [ $# -eq 1 ] || usage
-        CONTAINER=$(docker start $1)
+        RES=$(docker start $1)
+        CONTAINER=$(container_id $1)
         create_bridge
         get_ipam_cidr $CONTAINER
         with_container_netns_or_die $CONTAINER attach $CIDR_ARGS
         when_dns_running with_container_fqdn $CONTAINER put_dns_fqdn $CIDR_ARGS
-        echo $CONTAINER
+        echo $RES
```

Comment:
If we expected to get the full hex ID back, why should we not do the same for our users?